django/views/decorators/http.py

```diff
@@ -105,8 +124,8 @@ def get_last_modified():
                 last_modified=res_last_modified,
             )
 
-            if response is None:
-                response = func(request, *args, **kwargs)
+            # if response is None:
```

Comment:
Commenting this out didn't result in any failed tests on my local machine. I am keeping it commented out to see if it breaks on the CI.